manticore/backend/base.py

```diff
@@ -190,7 +190,7 @@ def sql_flush(self, style, tables, *, reset_sequences=False,
             sql = []
             for table in tables:
                 table = self.mark_table_name(table)
-                sql.append('%s %s;' % (
+                sql.append('%s %s' % (
```

Comment:
Because of this, the Manticore database above version 6.0.0 did not work.

Reply:
That's strange; at logical level queries are already separated from each other as in returned `list`, so it's meaningless.
I would like to get more detail explanation, why `;` is needed also.

My guess is that `sql` is later joined with separator other than `;` which leads to an error.
I propose to debug how this works for mysql database backend.

Reply:
Locally it works with and without `;`, but only without `;` it works here.

Reply:
My miss, `;` is being removed in this fix. Am i right? 
If so, no questions for code without `;`.

Reply:
Yes, `;` is being removed in this fix.

Reply:
Yes,` ;` is being removed in this fix.